### ISSUE TITLE
Generating "export class" without module/namespace

### ISSUE THREAD

**Issue by eommund** [reinforced/Reinforced.Typings#9]
When generating ts classes with RT, they are either generated within a module as "export class" or without a module (IncludeNamespace=false) - but then without the export keyword. 

I miss the option of generating "export class"es  without the module/namespace. 

The Angular2 team constantly uses "export class" without the module in their examples (for example https://angular.io/docs/ts/latest/guide/displaying-data.html), this leads me to believe that it would be valid to have an export class without a module.

**Comment by pavel-b-novikov** [comment]
Oh, that is what you are talking about. 
Well, I will consider latest TS changes and will embed support for them to RT soon (most likely next week). Please be patient.

**Comment by denimaaark** [comment]
Please can i ask about this request, is it maybe resolved ?

**Comment by pavel-b-novikov** [comment]
@denimaaark Still pending. Having too much job.

**Comment by veb82** [comment]
I miss this as well. I made this workaround you guys can use until this is implemented.
```
internal class ReinforcedTypingsExportClassCodeGenerator: ClassCodeGenerator
    {
        public override RtClass GenerateNode(Type element, RtClass result, TypeResolver resolver)
        {
            var rtClass= base.GenerateNode(element, result, resolver);
            Context.Location.CurrentModule.CompilationUnits.Add(new RtRaw("export"));
            return rtClass;
        }
```
then use it like 
`[TsClass(IncludeNamespace = false, CodeGeneratorType = typeof(ReinforcedTypingsExportClassCodeGenerator))]`

**Comment by pavel-b-novikov** [comment]
Thanks @veb82 
Could you please provide this one to gist?

**Comment by veb82** [comment]
@pavel-b-novikov https://gist.github.com/veb82/9017e78bb1c0fd2f9674e31851be8c2a

thanks

**Comment by gantrior** [comment]
+1

**Comment by pavel1269** [comment]
+1

**Comment by mulgurul** [comment]
+1, allow export on modules as attribute

**Comment by konrad-powazka** [comment]
+1

**Comment by AndreyTsvetkov** [comment]
+1

**Comment by pavel-b-novikov** [comment]
Oh..

**Comment by Silthus** [comment]
+1

**Comment by pavel-b-novikov** [comment]
Ok, guys. Good news. 
In version 1.3 there is modules export implemented (expecially for --module option). 
Now I'm writing tests for it, but actually it is almost done. 

You can begin dance right now.

**Comment by pavel-b-novikov** [comment]
@AndreyTsvetkov Please test modules mode

**Comment by pavel-b-novikov** [comment]
Fixed in 1.3.0
